test/funcfile-docker-rt-tests/fn_test.go

```diff
@@ -0,0 +1,154 @@
+package tests
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"testing"
+)
+
+func TestDockerRuntime(t *testing.T) {
+	testname := "TestDockerRuntime"
+	testfiles := []string{"Dockerfile", "func.go"}
+
+	currdir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("ERROR: Failed to get current directory: err: %v", err)
+	}
+	testdir, err := ioutil.TempDir("", "cli_test_funcfile-docker-rt_"+testname)
+	if err != nil {
+		t.Fatalf("ERROR: Failed to make tmp test directory: err: %v", err)
+	}
+	defer cleanup(t, currdir, testdir)
+	fnTestBin := setupTestFiles(t, testname, currdir, testdir, testfiles)
+	runFnInit(t, testname, fnTestBin)
+	runFnBuild(t, testname, fnTestBin)
+	runFnRun(t, testname, fnTestBin)
+	mylog(t, "INFO", testname, testname+" SUCCESS")
+}
+
+func TestDockerRuntimeNoDockerfile(t *testing.T) {
+	testname := "TestDockerRuntimeNoDockerfile"
+	testfiles := []string{"func.yaml", "func.go"}
+
+	currdir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("ERROR: Failed to get current directory: err: %v", err)
+	}
+	testdir, err := ioutil.TempDir("", "cli_test_funcfile-docker-rt_"+testname)
+	if err != nil {
+		t.Fatalf("ERROR: Failed to make tmp test directory: err: %v", err)
+	}
+	defer cleanup(t, currdir, testdir)
+	fnTestBin := setupTestFiles(t, testname, currdir, testdir, testfiles)
+	runFnBuildNoDockerfile(t, testname, fnTestBin)
+	mylog(t, "INFO", testname, testname+" SUCCESS")
+}
+
+func cleanup(t *testing.T, currdir, testdir string) {
+	err := os.Chdir(currdir)
+	if err != nil {
+		t.Fatalf("ERROR: Failed to cd "+currdir+" directory: err: %v", err)
```

Comment:
`t.Fatalf` is a formatting function, so you just need to do the following:

`t.Fatalf("ERROR: Failed to cd %v directory: err: %v", curdir, err)`

you might fix this across other calls.

Reply:
haven't really pay close attention to the test logging.  Thanks.  updated